src/tests/p11test/p11test_case_readonly.c

```diff
@@ -515,7 +562,7 @@ int verify_message(test_cert_t *o, token_info_t *info, CK_BYTE *message,
 	CK_RV rv;
 	CK_FUNCTION_LIST_PTR fp = info->function_pointer;
 	CK_MECHANISM sign_mechanism = { mech->mech, NULL_PTR, 0 };
-	static int verify_support = 1;
+	static int verify_support = 0;
```

Comment:
I think this should not be in the final version

Reply:
Fixed